keymapviz/keyboards/planck/layout_editor/grid.json

```diff
@@ -0,0 +1,63 @@
+[
```

Comment:
The files layout_editor/*.json also needs to have the legend replaced with `{}`. (as well as `ascii_art`).

It looks like the following.

https://github.com/yskoht/keymapviz/blob/develop/keymapviz/keyboards/lets_split/layout_editor/default.json